src/main/clojure/clara/tools/inspect.clj

```diff
@@ -15,8 +15,10 @@
 ;; This is derived from the Rete-style tokens, but this token
 ;; is designed to propagate all context needed to easily inspect
 ;; the state of rules.
-(s/defrecord Explanation [matches :- [[(s/one s/Any "fact") (s/one schema/Condition "condition")]] ; Fact, condition tuples
-                           bindings :- {s/Keyword s/Any}]) ; Bound variables
+(s/defrecord Explanation [matches :- [{:result s/Any
```

Comment:
Unfortunately, I don't see a name that really fits well with both the accumulator and non-accumulator cases.  It is probably better to go with a name that fits the majority non-accumulator case better.  I'll change this to :fact and elaborate more in documentation on the schema.